pdk/1.0/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -0,0 +1,41 @@
+= Compiling Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+You can compile the custom policy project after you complete all the steps in xref:policies-pdk-create-project.adoc[].
+
+You must compile the custom policy before you can deploy it to a Flex Gateway.
+
+After you compile your project, test your policy with the following tutorials:
+
+* xref:policies-pdk-debug-local.adoc[]
+* xref:policies-pdk-apply-policies.adoc[]
+
+== Before You Begin
+
+Ensure you have completed all the steps in xref:policies-pdk-create-project.adoc[].
+
+== Build Command
+
+To compile the policy, run the `build` command from the policy's root folder:
+
+[source,ssh]
+----
+make build
+----
+
+The `build` command preforms the following actions:
+
+. Runs the `build-asset-files`.
+. Runs the `build` command to compile the policy to a `WebAssembly` binary file.
+. Generates additional configuration files that aren't generated by the `build-asset-files` command and are required to execute the policy. 
```

Comment:
```suggestion
. Generates additional configuration files that are not generated by the `build-asset-files` command and are required to execute the policy
```
Please check with the style guide about using contractions. IIRC we should not use them, but I could be wrong.